src/tests/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -131,6 +133,20 @@ public void testVCFHeaderSampleRenamingSitesOnlyVCF() throws Exception {
         final VCFHeader header = (VCFHeader) codec.readHeader(vcfIterator).getHeaderValue();
     }
 
+    @Test
+    public void testVCFHeaderAddMetaDataLine() {
+        File input = new File("testdata/htsjdk/variant/ex2.vcf");
+
+        VCFFileReader reader = new VCFFileReader(input, false);
+        VCFHeader header = reader.getFileHeader();
+
+        VCFInfoHeaderLine newInfoField = new VCFInfoHeaderLine("test", VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "test info field");
+        header.addMetaDataLine(newInfoField);
+
+        // getting the sequence dictionary was failing due to duplicating contigs in issue #214,
+        // as long as we have no exception then we pass this test
+        header.getSequenceDictionary();
```

Comment:
Add an explicit assertion that the number of contig lines in the header (`header.getContigLines().size()`) is the same before and after the call to `addMetaDataLine()`